### ISSUE TITLE
Q: Using libtraceevent for rasdaemon, does it interfere with other ftrace users?

### ISSUE THREAD

**Issue by vt-alt** [mchehab/rasdaemon#173]
rasdaemon is using `libtraceevent`, but does it conflict or interfere with other ftrace users?

**Comment by mchehab** [comment]
I don't think so. Btw, rasdaemon was one of the first users of `libtraceevent` (besides ftrace itself): By the time I wrote it, there weren't even a separate package for it.

Anyway, as this is not really an issue, I'm closing it.
